astropy/io/fits/hdu/image.py

```diff
@@ -33,17 +37,25 @@ class _ImageBaseHDU(_ValidHDU):
     """
 
     standard_keyword_comments = {
-        'SIMPLE': 'conforms to FITS standard',
-        'XTENSION': 'Image extension',
-        'BITPIX': 'array data type',
-        'NAXIS': 'number of array dimensions',
-        'GROUPS': 'has groups',
-        'PCOUNT': 'number of parameters',
-        'GCOUNT': 'number of groups'
+        "SIMPLE": "conforms to FITS standard",
+        "XTENSION": "Image extension",
+        "BITPIX": "array data type",
+        "NAXIS": "number of array dimensions",
+        "GROUPS": "has groups",
+        "PCOUNT": "number of parameters",
+        "GCOUNT": "number of groups",
```

Comment:
Better to avoid stylistic changes in this PR which will already have a lot of changes (actually it seems you reformatted the whole file, please revert this).

Reply:
oops. that out have been a [black](https://pypi.org/project/black/) thing. Thanks for finding it.

Reply:
I would love to just run [black](https://pypi.org/project/black/) on all of astropy and PR. But that would be VERY controversial. 
Also add a big ``.pre-commit-config.yaml`` with black, pep8, RST-style checks, private-key detector, pragma checker, etc. There are so many nice style and security enforcers available.

Reply:
Yes, `black` would be very controversial!! But some of the others much less so. In fact, we had a PR for pre-commit but it was abandoned - #9890. I think the trick would be to take small steps only...